Makefile

```diff
@@ -1,7 +1,7 @@
 all: thrift
 
-THRIFT=thrift1
-THRIFT_OPTS=-strict -gen py:utf8strings,slots,new_style
+THRIFT=thrift
+THRIFT_OPTS=-strict -gen py:slots
```

Comment:
note: utf8strings and new_style are now the default in more modern thrift compilers and we get a warning about the option going away if we try to set them explicitly